todo_app/ViewModel.py

```diff
@@ -0,0 +1,24 @@
+
+
+class ViewModel :
+     def __init__(self, alltodoitems) :
+         self._todo = alltodoitems
+
+     @property
+     def my_item(self) :
```

Comment:
"my_item" sounds like a single item, but it's actually the full list of to do items.